### ISSUE TITLE
Use GeographicLib crate to calculate coordinates in geographic coordinates

### ISSUE THREAD

**Issue by Robinlovelace** [zonebuilders/zonebuilder-rust#33]
From https://github.com/georust/geographiclib-rs

```
use geographiclib_rs::{Geodesic, DirectGeodesic};

let g = Geodesic::wgs84();
let jfk_lat = 40.64;
let jfk_lon = -73.78;
let northeast_azimuth = 45.0;

let (lat, lon, az) = g.direct(jfk_lat, jfk_lon, northeast_azimuth, 10e6);
```

**Comment by Robinlovelace** [comment]
Partly supersedes #31.

**Comment by Robinlovelace** [comment]
Wow. That was way easier than expected! Result already: 

![image](https://user-images.githubusercontent.com/1825120/116782635-5399f080-aa82-11eb-8dc3-70184860243e.png)

See the example branch for this implemented for circles :tada: 

Now just need to do the same for the polygon building function and possibly add an argument `projected` which is `true` or `false`. I suggest `false` is the default.

**Comment by Robinlovelace** [comment]
Fixed in #34
